catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -682,15 +682,27 @@ const WorkDetails: FunctionComponent<Props> = ({ work }: Props) => {
         <WorkDetailsSection headingText="Subjects">
           <WorkDetailsTags
             tags={work.subjects.map(s => {
-              return {
-                textParts: s.concepts.map(c => c.label),
-                linkAttributes: worksLink(
-                  {
-                    'subjects.label': [s.label],
-                  },
-                  'work_details/subjects'
-                ),
-              };
+              return s.id
+                ? {
+                    textParts: ['💚', ...s.concepts.map(c => c.label)],
```

Comment:
This is a very simple idea meant to highlight identified concepts when browsing works. It should be behind a toggle before we put this on the site proper.